chia/util/keychain.py

```diff
@@ -203,6 +210,21 @@ def mnemonic_to_seed(mnemonic: str, passphrase: str) -> bytes:
     return seed
 
 
+def default_keychain_user() -> str:
+    return DEFAULT_USER
+
+
+def default_keychain_service() -> str:
+    return DEFAULT_SERVICE
+
+
+def get_private_key_user(user: str, index: int) -> str:
```

Comment:
why does this function name contain `private_key`, when it doesn't seem to have anything to do with any private key?

Reply:
This is a naming hold-over from the original keychain implementation (previously _get_private_key_user). The function returns the "name" of the private key in the key store. I could change it, but the name predates my work in this code, and the name seems reasonable, no?

Reply:
I see, so it's more like: `get_user_for_private_key()`?
Yeah, I don't think it should be renamed in this PR